tox.ini

```diff
@@ -95,12 +95,14 @@ commands =
     lint: pylint --rcfile=tests/pyproject.toml tests
     lint: pydocstyle lms tests bin
     lint: pycodestyle lms tests bin
-    tests: python -m pytest --cov --cov-report= --cov-fail-under=0 --numprocesses logical --failed-first --new-first --no-header --quiet {posargs:tests/unit/}
+    {tests,functests}: python3 -m lms.scripts.init_db --delete --create
+    tests: python -m pytest --cov --cov-report= --cov-fail-under=0 --numprocesses logical --dist loadgroup --failed-first --new-first --no-header --quiet {posargs:tests/unit/}
```

Comment:
The `--dist loadgroup` makes the `@pytest.mark.xdist_group()`'s work for defining groups of tests that should all be passed to the same runner. It's already used when running the tests in parallel in h so I was gonna have to add it to the cookiecutter anyway.